.vscode/README.md

```diff
@@ -0,0 +1,39 @@
+# CSpell Extension Configuration
+
+## Setup for testing changes locally before submit
+
+1. Install VS Code.
+2. Install the Code Spell Checker by Street Side Software.
+This will allow testing of the changes locally with minimal effort. Literally, you'll just need to load a file and anything it flags will be underlined in blue. The configuration definition can be found at <https://cspell.org/configuration/>
+
+## Words and word boundaries
```

Comment:
This section and the sections below it should be in https://github.com/Azure/azure-sdk-tools/blob/main/doc/common/spellcheck.md so that other languages can benefit from the knowledge here. 

This file should provide the basic local testing instructions and link off to https://github.com/Azure/azure-sdk-tools/blob/main/doc/common/spellcheck.md for the deeper dive into how spell checking works.

Reply:
@danieljurek, I added the link to common doc in my readme. I'm going to leave it to you to update the common one with the rules, from this readme, that you think apply to all languages.